src/fio_planet/features.py

```diff
@@ -51,6 +52,35 @@ def __getitem__(self, key):
             )
 
 
+def area(geom: Union[BaseGeometry, BaseMultipartGeometry], projected=False) -> float:
+    """The cartesian or projected area of a geometry.
+
+    The default is to return a unitless Cartesian area. If reproject is True,
+    the input geometry will be reprojected to the EASE grid system before
+    computing its area and the value will have units of m**2.
+
+    Parameters
+    ----------
+    geom : a shapely geometry object
+    projected : bool, optional
+        If True, reproject to EASE grid system with units of m**2. Else
+        return a unitless Cartesian area.
```

Comment:
I'm open to making projected the default. Yeah? No?

Reply:
Doing it.

Reply:
From personal experience, I can't see why I would want it to default to anything other than m**2.

Reply:
I heard another vote for projected by default. Gonna merge this.